hunts/chart_utils.py

```diff
@@ -0,0 +1,41 @@
+from django.utils import timezone
+
+from .models import Hunt
+from puzzles.models import Puzzle
+
+
+def can_use_chart(hunt):
+    if not hunt.start_time or timezone.now() <= hunt.start_time:
```

Comment:
Can we use first puzzle creation time as a proxy when `hunt.start_time` doesn't exist?